sycl/include/CL/sycl/buffer.hpp

```diff
@@ -181,9 +181,8 @@ class buffer {
       : Range{0} {
 
     size_t BufSize = 0;
-    PI_CALL(detail::RT::piMemGetInfo,
-        detail::pi::cast<detail::RT::PiMem>(MemObject), CL_MEM_SIZE,
-        sizeof(size_t), &BufSize, nullptr);
+    PI_CALL(piMemGetInfo, detail::pi::cast<detail::RT::PiMem>(MemObject),
+            CL_MEM_SIZE, sizeof(size_t), &BufSize, nullptr);
```

Comment:
Only noticing this now - this somehow reminds me of Lisp :smile:

Reply:
I have never used Lisp. Are you talking about use of comma after the api name?
I have the call interface changed in #843 . Do you think that works?
Maybe I can close this PR and merge the two PRs together. 
Let me know what you think.

Reply:
Oh, sorry for the confusion and the extra work that I now caused - I meant "Lisp" as a positive thing :crying_cat_face: 

Both approaches seem ok to me, the one in this PR seems to require less state in the trace class though.